src/infra/core/src/ingest/fetch_service.rs

```diff
@@ -313,6 +313,7 @@ impl FetchService {
                 if #[cfg(unix)] {
                     self.container_runtime
                         .run_attached(&fetch.image)
+                        .container_name_prefix("kamu-permissions-")
```

Comment:
Optional: rename `container_name_prefix()` to `random_container_name_with_prefix()` (or similar)

Reply:
+1 to better name and hiding name generation fully in `container-runtime` crate

we should also provide a similar method for creating randomized networks